modelci/types/vo/model_vo.py

```diff
@@ -246,10 +250,25 @@ class ModelIn(BaseModel):
     dataset: str
     metric: Dict[Metric, float]
     task: Task
+    parent_model_id: str
     inputs: List[IOShapeVO]
     outputs: List[IOShapeVO]
     architecture: str
     framework: Framework
     version: int
     convert: bool
     profile: bool
+
+class TrainerConfig(BaseModel):
```

Comment:
Move this class to `trainer.py`. This class has nothing to do with the model object.